markpbaggett/exodus
# Correct Xpaths for contributor, utk_contributor, creator, utk_creator…

…. Remove  get_creators.

## What does this Pull Request do?

Correct Xpaths were never entered for collection properties contributor, utk_contributor, creator, and utk_creator. They have both been added in the [collection tab on the Vendor MAP](https://docs.google.com/spreadsheets/d/1_0QVbQU_wj3ITUih5dGPGkWHN0QyhGO9hKSf6rXwKPc/edit#gid=365080784&range=G4) and in this code. As I didn't see anything that was actionable from "get_creators", I also removed that from the code.

## How should this be tested?

Run it against collection records and see if creators/contributors with and without URIs are correctly mapped. Spot check for simple typos in roleTerm values.

## Interested parties

@markpbaggett
